src/x/mmap/mmap_linux.go

```diff
@@ -109,3 +109,8 @@ func Munmap(b []byte) error {
 
 	return nil
 }
+
+// Free mmapped memory.
```

Comment:
Maybe we should add a note about the different behavior with MADV_DONTNEED based on whether the mmap is an anonymous or file based mmap?